.devcontainer/createCommand.sh

```diff
@@ -19,5 +24,11 @@ rm -f .env
 echo "PYTHONPATH=""$path"":$PYTHONPATH" >> .env
 echo "MYPYPATH=""$path"":$MYPYPATH" >> .env
 
+echo "Setting up git safe directory"
+git config --global --add safe.directory /workspaces/content
+
 echo "Setting up content dependencies"
-.hooks/bootstrap
\ No newline at end of file
+.hooks/bootstrap
+
+echo "Run demisto-sdk pre-commit to cache dependencies"
+poetry run demisto-sdk pre-commit > /dev/null 2>&1 || true
```

Comment:
```suggestion
poetry run demisto-sdk pre-commit 2>&1 >/dev/null || true
```
redirects is read from left to right. so first point stderr to stdout, then point stdout to null.

why not point it to a log file for forensic if needed?

Reply:
We don't care about the output of the pre-commit. We do it, so the pre-commit dependencies will be cached when running `demisto-sdk pre-commit` in the future.